gbi.h

```diff
@@ -1126,38 +1130,49 @@ typedef union {
     long long int force_structure_alignment[2];
 } Vp;
 
-/*
+/**
  * Light structure.
  *
  * Note: the weird order is for the DMEM alignment benefit of
  * the microcode.
  */
-
 typedef struct {
-    unsigned char col[3];   /* diffuse light color (rgb) */
-    unsigned char type;     /* formerly pad1; MUST SET TO 0 to indicate directional light */
-    unsigned char colc[3];  /* copy of diffuse light color (rgb) */
+    unsigned char col[3];   /** diffuse light color (rgb) */
+    unsigned char type;     /** formerly pad1; MUST SET TO 0 to indicate directional light */
+    unsigned char colc[3];  /** copy of diffuse light color (rgb) */
     char          pad2;
-    signed char   dir[3];   /* direction of light (normalized) */
+    signed char   dir[3];   /** direction of light (normalized) */
     char          pad3;
     char          pad4[3];
-    unsigned char size;     /* For specular only; reasonable values are 1-4 */
+    unsigned char size;     /** For specular only; reasonable values are 1-4 */
 } Light_t;
 
+/**
+ * Light structure.
+ *
+ * Note: the weird order is for the DMEM alignment benefit of
+ * the microcode.
+ */
 typedef struct {
-    unsigned char col[3];   /* point light color (rgb) */
-    unsigned char kc;       /* point light enable flag (> 0) & constant attenuation Kc */
-    unsigned char colc[3];  /* copy of point light color (rgb) */
-    unsigned char kl;       /* linear attenuation Kl */
-    short pos[3];           /* light position x, y, z in world space */
-    unsigned char kq;       /* quadratic attenuation Kq */
-    unsigned char size;     /* For specular only; reasonable values are 1-4 */
+    unsigned char col[3];   /** point light color (rgb) */
+    unsigned char kc;       /** point light enable flag (> 0) & constant attenuation Kc */
+    unsigned char colc[3];  /** copy of point light color (rgb) */
+    unsigned char kl;       /** linear attenuation Kl */
+    short pos[3];           /** light position x, y, z in world space */
+    unsigned char kq;       /** quadratic attenuation Kq */
+    unsigned char size;     /** For specular only; reasonable values are 1-4 */
 } PosLight_t;
 
+/**
+ * Light structure.
+ *
+ * Note: the weird order is for the DMEM alignment benefit of
+ * the microcode.
+ */
```

Comment:
Would you mind changing this and all the times a comment has been copied to use `@see other_thing` instead of copying the comments? With the copied comments, if I make changes in the future I have to copy them again, and it makes things a lot longer.

Reply:
I change to use `@copydetails other_thing` instead to avoid have bunch of link and so lose track of what we search in docs

Reply:
(I use copydetails instead of copydocs because copydocs also copy the brief and I think in brief just put if it's a macro for static display or for add to a displaylisthead)